pom.xml

```diff
@@ -58,15 +58,35 @@
         <sortpom-maven-plugin.version>2.8.0</sortpom-maven-plugin.version>
         <spring-security.version>5.3.8.RELEASE</spring-security.version>
         <springfox.version>3.0.0</springfox.version>
-        <tomcat.version>9.0.58</tomcat.version>
+        <tomcat.version>9.0.63</tomcat.version>
         <wiremock.version>2.1.3.RELEASE</wiremock.version>
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.13.3</version>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.13.3</version>
```

Comment:
Multiple references of version 2.13.3. have a variable similar to other versions